docs/source/jit_language_reference_statements.rst

```diff
@@ -0,0 +1,389 @@
+    `starred_expression <https://docs.python.org/3/reference/expressions.html#grammar-token-starred-expression>`
+
+* `yield expression` is not supported in Torchscript.
+
+Augmented assignment statements:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    augmented_assignment_stmt ::=  augtarget augop (expression_list)
+    augtarget ::=  identifier | attributeref | subscription
+    augop ::=  "+=" | "-=" | "*=" | "/=" | "//=" | "%=" |
+                "**="| ">>=" | "<<=" | "&=" | "^=" | "|="
+    `augtarget <https://docs.python.org/3/reference/simple_stmts.html#grammar-token-augtarget>`
+    `augop <https://docs.python.org/3/reference/simple_stmts.html#grammar-token-augop>`
+    `identifier <https://docs.python.org/3/reference/lexical_analysis.html#grammar-token-identifier>`
+    `attributeref <https://docs.python.org/3/reference/expressions.html#grammar-token-attributeref>`
+    `subscription <https://docs.python.org/3/reference/expressions.html#grammar-token-subscription>`
+    `expression_list <https://docs.python.org/3/reference/expressions.html#grammar-token-expression-list>`
+
+* `yield expression` is not supported in Torchscript.
+
+Annotated assignment statements:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    annotated_assignment_stmt ::=  augtarget ":" expression
+                               ["=" (starred_expression)]
+    `augtarget <https://docs.python.org/3/reference/simple_stmts.html#grammar-token-augtarget>`
+    `expression <https://docs.python.org/3/reference/expressions.html#grammar-token-expression>`
+    `starred_expression <https://docs.python.org/3/reference/expressions.html#grammar-token-starred-expression>`
+
+* `yield expression` is not supported in Torchscript.
+
+The ``raise`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    raise_stmt ::=  "raise" [expression ["from" expression]]
+    `expression <https://docs.python.org/3/reference/expressions.html#grammar-token-expression>`
+
+* Raise statements in TorchScript do not support ``try\finally``.
+
+The ``assert`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    assert_stmt ::=  "assert" expression ["," expression]
+    `expression <https://docs.python.org/3/reference/expressions.html#grammar-token-expression>`
+
+* Assert statements in TorchScript do not support ``try\finally``.
+
+The ``return`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    return_stmt ::=  "return" [expression_list]
+    `expression_list <https://docs.python.org/3/reference/expressions.html#grammar-token-expression-list>`
+
+* Return statements in TorchScript do not support ``try\except\finally``.
+
+The ``del`` statement:
+^^^^^^^^^^^^^^^^^^^^^^
+::
+    del_stmt ::=  "del" target_list
+    `target_list <https://docs.python.org/3/reference/simple_stmts.html#grammar-token-target-list>`
+
+The ``pass`` statement:
+^^^^^^^^^^^^^^^^^^^^^^
+::
+    pass_stmt ::= "pass"
+
+The ``print`` statement:
+^^^^^^^^^^^^^^^^^^^^^^^^
+::
+    print_stmt ::= "print"
```

Comment:
This seems incorrect or incomplete. Isn't `print("hello")` a valid statement?

Reply:
This seems strange to me as well, I thought one thing that Python3 changes compared to Python2 is that `print` is now a proper builtin function rather than a statement?

Reply:
Agreed,. Print is a python builtin function. removed it now